src/libpgmoneta/prometheus.c

```diff
@@ -326,6 +328,51 @@ home_page(int client_fd)
    data = pgmoneta_append(data, "  <h2>pgmoneta_wal_shipping_total_space</h2>\n");
    data = pgmoneta_append(data, "  The total disk space for the WAL shipping directory of a server\n");
    data = pgmoneta_append(data, "  <p>\n");
+   data = pgmoneta_append(data, "  <h2>pgmoneta_server_timeline</h2>\n");
+   data = pgmoneta_append(data, "  The current timeline a server is on\n");
+   data = pgmoneta_append(data, "  <table border=\"1\">\n");
+   data = pgmoneta_append(data, "    <tbody>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>name</td>\n");
+   data = pgmoneta_append(data, "        <td>The identifier for the server</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "    </tbody>\n");
+   data = pgmoneta_append(data, "  </table>\n");
+   data = pgmoneta_append(data, "  <p>\n");
+   data = pgmoneta_append(data, "  <h2>pgmoneta_server_parent_tli</h2>\n");
+   data = pgmoneta_append(data, "  The parent timeline of a timeline on a server\n");
+   data = pgmoneta_append(data, "  <table border=\"1\">\n");
+   data = pgmoneta_append(data, "    <tbody>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>name</td>\n");
+   data = pgmoneta_append(data, "        <td>The identifier for the server</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>tli</td>\n");
+   data = pgmoneta_append(data, "        <td>The current/previous timeline ID for the server</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "    </tbody>\n");
+   data = pgmoneta_append(data, "  </table>\n");
+   data = pgmoneta_append(data, "  <p>\n");
+   data = pgmoneta_append(data, "  <h2>pgmoneta_server_timeline_switchpos</h2>\n");
+   data = pgmoneta_append(data, "  The xlog switch position in decimal of a timeline on a server\n");
+   data = pgmoneta_append(data, "  <table border=\"1\">\n");
+   data = pgmoneta_append(data, "    <tbody>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>name</td>\n");
+   data = pgmoneta_append(data, "        <td>The identifier for the server</td>\n");
+   data = pgmoneta_append(data, "      </tr>\n");
+   data = pgmoneta_append(data, "      <tr>\n");
+   data = pgmoneta_append(data, "        <td>tli</td>\n");
+   data = pgmoneta_append(data, "        <td>The current/previous timeline ID for the server</td>\n");
```

Comment:
It should always be `'current`, right ?

Reply:
It depends, we also know the parents of previous timelines, i.e. what timeline each timeline branched off from, so I showed them all... If you feel `current` is enough I can change that.